getting-started/deploy-to-dabl/6_upload_artifacts.md

```diff
@@ -0,0 +1,25 @@
+Time to upload your artifacts to your new ledger!
+
+First download the artifacts you built in the previous steps:
+
+- [Download create-daml-app.dar](https://[[HOST_SUBDOMAIN]]-8080-[[KATACODA_HOST]].environments.katacoda.com/create-daml-app.dar)
+
+- [Download create-daml-app-ui.zip](https://[[HOST_SUBDOMAIN]]-8080-[[KATACODA_HOST]].environments.katacoda.com/create-daml-app-ui.zip)
+
+Then click on your newly created ledger and under the _"App Artifacts"_ tab click on _"Upload .dar"_ and upload the `create-daml-app.dar` you just downloaded.
+
+![Upload dar](/daml/courses/getting-started/deploy-to-dabl/assets/project-dabl-upload-dar.gif)
```

Comment:
@robin-da recently figured out that you can reference assets with a relative path just with `asset/asset-name.extension. See his example [here](https://github.com/digital-asset/daml-katacoda/pull/48/files#diff-9c5684b383b3056a3eaa94475e5a90d9).

Reply:
Doesn't work in embedded mode.

Reply:
nvm, Katacoda seem to have fixed the issue that was preventing that working originally.

Reply:
will use the relative paths